visidata/pyobj.py

```diff
@@ -1,6 +1,13 @@
+from functools import singledispatch
+
 from visidata import *
 
 option('visibility', 0, 'visibility level')
+isNull = lambda v: any((
```

Comment:
isNull must be created at runtime or else changing `null_value` has no effect.  The idiom in VisiData is to call `isNullFunc()` to return `isNull` just before using it (but only once if using it in a loop, because getting an option value is expensive); that way it should be handled consistently across all cases considering nulls

Reply:
Thanks @saulpw, good call! I had tried to use `isNullFunc` but ran into issues trying to null-check columns with non-hashable values like lists. I was trying to keep this change local, but now that you mention it maybe I'll try tweaking that `isNullFunc` to cover this case without changing its intent.